platform/podjs/README.md

```diff
@@ -29,12 +29,102 @@ soon.
+1.  In order to point to `podjs` from the polypoly-eu repository,
+    add as `development dependencies` on your `package.json` the following:
+
+      <code>
+
+         "devDependencies": {
+            "@polypoly-eu/podjs": "file:../../platform/podjs",
+         },
+
+      </code>
+
+2.  We use the `rollup` module bundler to build our features all together via the script
+
+    `$ rollup -c rollup.config.json`
+
+    which runs `rollup` and takes the `./src` directory as input and
+    puts the result in a `dist` folder. Those configurations need to be defined
+    in the `rollup.config.js` file.
+
+    We basically rely on our in-house `rollup-plugin-copy-watch` plugin
+    on top of the `Rollup` one which serves to copy files and folders with glob support
+    and also offers an additional watch to other sources than just Rollup's bundle content.
+    This is also needed to be installed as a dev dependency:
+
+    `$ npm i --save-dev ../../dev-utils/rollup-plugin-copy-watch`
+
+    So, we end up with a `package.json` file like this:
+
+         "devDependencies": {
+            "@polypoly-eu/rollup-plugin-copy-watch": "file:../../dev-utils/rollup-plugin-copy-watch",
+         },
+
+    So the feature uses the `@polypoly-eu/rollup-plugin-copy-watch"` plugin to watch the copied files.
+
+3.  Next step, you need to define your `rollup.config.js` at the same level as your `package.json`.
+
+    There you need to have the `copy` (of our `"@polypoly-eu/rollup-plugin-copy-watch"`) under the `plugins` section with the `targets`
+    setup for `src`, which in this case it's the `"node_modules/@polypoly-eu/podjs/dist/pod.js"`, and `dest` where it will be ended to (e.g. `dist`).
+    So, it looks like:
+
+```
+   import copy from "@polypoly-eu/rollup-plugin-copy-watch";
+
+   export default {
+      input: "src/your-input-file.js",    // point to your source file
+      output: {
+         file: "dist/myfile.js", // point to your output file
+         format: "iife",     // the format of your output file, for now only iife is supported
+         globals: {
+               // where you define all the external packages you want to use
+               pod: "pod",
+            },
+         }
+      },
+      plugins: [
+         ...
+         copy({
+            targets: [
+            {
+               src: [
+                  "node_modules/@polypoly-eu/podjs/dist/pod.js",
+                  ...
+               ],
+               dest: "dist",
+            },
+         ]
+         }),
+      ]
+   }
+```
+
+This way you have copied successfully the `pod.js` file from the `@polypoly-eu/podjs` module, and you can use it in your feature.
+
+4.  So, to compile the input files and run the previous configurations,
+    you need to define a `build` script in the `package.json` file,
+    which will run the `rollup` command:
+
+```
+   {
+      ...
+      "scripts": {
+         "build": "shx rm -rf dist && tsc && rollup -c rollup.config.js",
```

Comment:
Rarely the features are written in TypeScript; instead of using it as the only example, a couple of them with or without TS might be better.
You might need to clarify that `shx` needs to be installed; maybe a minimal dep configuration for a feature could be included before this (with rollup, our plugin, podjs, shx, and anything else that might be needed)

Reply:
Sorry, why is this resolved? The features include directly the generated pod.js file, so they don't need to be written in Typescript; there's only one written in TS, as a matter of fact. Typescript is not listed as a requirement (for the feature) above, so have you considered not using it as the default alternative and just skip that part? Alternatively, you can specify that `tsc` is only needed in case the feature is written in TS, and an alternative command line in that case.